Tasks/Ananich Artem/Task_6/dz_6.1.py

```diff
@@ -0,0 +1,37 @@
+# 1. Написать рекурсивную функцию для вычисления суммы всех элементов вложенных (любая глубина) списков.
+# Пример списка (синтаксис Python): [1, 2, [2, 4, [[7, 8], 4, 6]]], сумма элементов - 34
+
+# 2. Написать рекурсивную функцию для вычисления n первых чисел Фибоначчи.
+# Примеры вызова: 
+# fib(5) -> 0,1,1,2,3
+# fib(10) -> 0,1,1,2,3,5,8,13,21,34
+
+
+#1
+l = [1, 2, [2, 4, [[7, 8], 4, 6]]]
+sum = 0
+def summ(lis):
+    for i in lis:
+        if not isinstance(i,list):#если i попадает не на список то мы начинаем суммирование
+            global sum #используем global чтоб сумма не обнулялась при рекурсивном вызове функции
```

Comment:
global is no good here, consider using an additional function parameter to store sum values